src/um/webauthn.rs

```diff
@@ -0,0 +1,221 @@
+pub const WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS_VERSION_1: DWORD = 1;
+pub const WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS_VERSION_2: DWORD = 2;
+pub const WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS_VERSION_3: DWORD = 3;
+pub const WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS_VERSION_4: DWORD = 4;
+pub const WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS_CURRENT_VERSION: DWORD =
+    WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS_VERSION_4;
+STRUCT!{struct WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS {
+    dwVersion: DWORD,
+    dwTimeoutMilliseconds: DWORD,
+    CredentialList: WEBAUTHN_CREDENTIALS,
+    Extensions: WEBAUTHN_EXTENSIONS,
+    dwAuthenticatorAttachment: DWORD,
+    dwUserVerificationRequirement: DWORD,
+    dwFlags: DWORD,
+    pwszU2fAppId: PCWSTR,
+    pbU2fAppId: *mut BOOL,
+    pCancellationId: *mut GUID,
+    pAllowCredentialList: PWEBAUTHN_CREDENTIAL_LIST,
+}}
+pub type PWEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS = *mut
+    WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS;
+pub type PCWEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS = *const
+    WEBAUTHN_AUTHENTICATOR_GET_ASSERTION_OPTIONS;
+pub const WEBAUTHN_CREDENTIAL_ATTESTATION_VERSION_1: DWORD = 1;
+pub const WEBAUTHN_CREDENTIAL_ATTESTATION_VERSION_2: DWORD = 2;
+pub const WEBAUTHN_CREDENTIAL_ATTESTATION_VERSION_3: DWORD = 3;
+pub const WEBAUTHN_CREDENTIAL_ATTESTATION_CURRENT_VERSION: DWORD =
+    WEBAUTHN_CREDENTIAL_ATTESTATION_VERSION_3;
+STRUCT!{struct WEBAUTHN_CREDENTIAL_ATTESTATION {
+    dwVersion: DWORD,
+    pwszFormatType: PCWSTR,
+    cbAuthenticatorData: DWORD,
+    pbAuthenticatorData: PBYTE,
+    cbAttestation: DWORD,
+    pbAttestation: PBYTE,
+    dwAttestationDecodeType: DWORD,
+    pvAttestationDecode: PVOID,
+    cbAttestationObject: DWORD,
+    pbAttestationObject: PBYTE,
+    cbCredentialId: DWORD,
+    pbCredentialId: PBYTE,
+    Extensions: WEBAUTHN_EXTENSIONS,
+    dwUsedTransport: DWORD,
+}}
+pub type PWEBAUTHN_CREDENTIAL_ATTESTATION = *mut WEBAUTHN_CREDENTIAL_ATTESTATION;
+pub type PCWEBAUTHN_CREDENTIAL_ATTESTATION = *const WEBAUTHN_CREDENTIAL_ATTESTATION;
+pub const WEBAUTHN_ASSERTION_CURRENT_VERSION: DWORD = 1;
+STRUCT!{struct WEBAUTHN_ASSERTION {
+    dwVersion: DWORD,
+    cbAuthenticatorData: DWORD,
+    pbAuthenticatorData: PBYTE,
+    cbSignature: DWORD,
+    pbSignature: PBYTE,
+    Credential: WEBAUTHN_CREDENTIAL,
+    cbUserId: DWORD,
+    pbUserId: PBYTE,
+}}
+pub type PWEBAUTHN_ASSERTION = *mut WEBAUTHN_ASSERTION;
+pub type PCWEBAUTHN_ASSERTION = *const WEBAUTHN_ASSERTION;
+extern "system" {
+    pub fn WebAuthNAuthenticatorMakeCredential(
+        hWnd: HWND,
+        pRpInformation: PCWEBAUTHN_RP_ENTITY_INFORMATION,
+        pUserInformation: PCWEBAUTHN_USER_ENTITY_INFORMATION,
+        pPubKeyCredParams: PCWEBAUTHN_COSE_CREDENTIAL_PARAMETERS,
+        pWebAuthNClientData: PCWEBAUTHN_CLIENT_DATA,
+        pWebAuthNMakeCredentialOPtions: PCWEBAUTHN_AUTHENTICATOR_MAKE_CREDENTIAL_OPTIONS,
```

Comment:
Nit: pWebAuthNMakeCredentialO**P**tions miscapitalized

Reply:
Fixed in 7dc2d8502925678e89c51c5779ff2868bbfcba7c.